source/extensions/common/wasm/context.cc

```diff
@@ -393,11 +393,9 @@ WasmResult serializeValue(Filters::Common::Expr::CelValue value, std::string* re
     auto size = proxy_wasm::PairsUtil::pairsSize(pairs);
     // prevent string inlining which violates byte alignment
     result->resize(std::max(size, static_cast<size_t>(30)));
-    if (!proxy_wasm::PairsUtil::marshalPairs(pairs, result->data(), size)) {
-      return WasmResult::SerializationFailure;
-    }
+    auto ok = proxy_wasm::PairsUtil::marshalPairs(pairs, result->data(), size);
     result->resize(size);
-    return WasmResult::Ok;
+    return ok ? WasmResult::Ok : WasmResult::SerializationFailure;
```

Comment:
Not fan of this kind of style change to improve coverage report artificially, also this change whether `result->resize(size);` is run.

Reply:
reverted.